Assets/Mirror/CompilerSymbols/PreprocessorDefine.cs

```diff
@@ -15,25 +15,7 @@ public static void AddDefineSymbols()
             HashSet<string> defines = new HashSet<string>(currentDefines.Split(';'))
             {
                 "MIRROR",
-                "MIRROR_1726_OR_NEWER",
-                "MIRROR_3_0_OR_NEWER",
-                "MIRROR_3_12_OR_NEWER",
-                "MIRROR_4_0_OR_NEWER",
-                "MIRROR_5_0_OR_NEWER",
-                "MIRROR_6_0_OR_NEWER",
-                "MIRROR_7_0_OR_NEWER",
-                "MIRROR_8_0_OR_NEWER",
-                "MIRROR_9_0_OR_NEWER",
-                "MIRROR_10_0_OR_NEWER",
-                "MIRROR_11_0_OR_NEWER",
-                "MIRROR_12_0_OR_NEWER",
-                "MIRROR_13_0_OR_NEWER",
-                "MIRROR_14_0_OR_NEWER",
-                "MIRROR_15_0_OR_NEWER",
-                "MIRROR_16_0_OR_NEWER",
-                "MIRROR_17_0_OR_NEWER",
-                "MIRROR_18_0_OR_NEWER",
-                "MIRROR_24_0_OR_NEWER",
```

Comment:
After talking with asset devs, let's keep 17, 18, and 24 in the list.

Reply:
Alright, adding those back into the list. I have no objections.